packages/@glimmer/runtime/lib/debug-render-tree.ts

```diff
@@ -181,7 +181,7 @@ export default class DebugRenderTreeImpl<TBucket extends object>
     let template = this.captureTemplate(node);
     let bounds = this.captureBounds(node);
     let children = this.captureRefs(refs);
-    return { id, type, name, args: reifyArgs(args), instance, template, bounds, children };
+    return { id, type, name, args: reifyArgsDebug(args), instance, template, bounds, children };
```

Comment:
Does this always happen? Do we want this to occur un production or dev only?

Reply:
i think its should be for both